orc8r/cloud/go/services/configurator/types.go

```diff
@@ -270,6 +284,72 @@ func (n NetworkEntitiesByTK) MultiFilter(types ...string) NetworkEntitiesByTK {
 	return filtered
 }
 
+// SerializedNetworkEntity is the same as NetworkEntity, but its config is
+// serialized.
+type SerializedNetworkEntity struct {
```

Comment:
I don't like that you've duplicated all the fields from `NetworkEntity` here. If we really need something like this, add an unexported `isDeserialized` field to `NetworkEntity` instead and add a `GetRawConfig` method to the struct or something like that. You can just pack the []byte into the interface{} type of Config

You'll probably need some `fromStorageProtoRaw` or something too, or maybe just a flag on the method.